Sources/RPListener.swift

```diff
@@ -67,7 +68,7 @@ public class RPListener: NSObject, XCTestObservation {
   }
     
   public func testBundleWillStart(_ testBundle: Bundle) {
-    let configuration = readConfiguration(from: testBundle)
+    self.configuration = readConfiguration(from: testBundle)
     
     guard configuration.shouldSendReport else {
```

Comment:
Using `guard` to test if some code should execute doesn't seem right to me, maybe it would make more sense to use `if` here? Something like
``` Swift
if configuration.shouldSendReport {
    reportingService = ReportingService(configuration: configuration)
    queue.async {
          do {
            try self.reportingService.startLaunch()
          }     catch let error {
            print(error)
          }
        }
    } else {
      print("Set 'YES' for 'PushTestDataToReportPortal' property in Info.plist if you want to put data to report portal")
}
```

would make more sense to me when reading this code. Same goes for other instances of `guard` used instead of `if` in this file.

Reply:
Was changed to if.